app/main.py

```diff
@@ -1,13 +1,23 @@
 def format_linter_error(error: dict) -> dict:
     # white your code here
```

Comment:
Please, remove the redundant comment

Reply:
i dont known how to do this i have no comments in pycharm